python/spacewalk/common/repo.py

```diff
@@ -315,11 +315,8 @@ def _has_valid_gpg_signature(self, uri: str, response=None) -> bool:
                         )
                     )
             else:
-                logging.error(
-                    # pylint: disable-next=logging-format-interpolation,consider-using-f-string
-                    "No release file found: '{}'. Raising GeneralRepoException.".format(
-                        uri
-                    )
+                logger.error(
+                    "No release file found: '%s'. Raising GeneralRepoException.", uri
```

Comment:
You removed the `Raising GeneralRepoException` from the previous logger, but kept it for the rest. Should we remove the rest as well? Personally, I prefer not having it in the logger message because:

a) it's redundant
b) it's not consistent, i.e. sometimes this ends in a full stop, other times in an exclamation mark
c) it could become obsolete very easily if someone were to change the actual exception (though unlikely)

But, if you'd prefer to keep it, I'm OK with that as well, in which case you might re-add it above :)

Reply:
Right, we can remove it everywhere :+1: